hack/build-and-push.sh

```diff
@@ -7,6 +7,20 @@ QUAY_ORG=redhat-appstudio-tekton-catalog
 SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 WORKDIR=$(mktemp -d --suffix "-$(basename "${BASH_SOURCE[0]}" .sh)")
 
+if ! command -v tash &> /dev/null; then
+  echo INFO: tash command is not available will download and use the latest version
```

Comment:
I agree, I did not want to end up troubleshooting and assisting folk in getting that latest version including the fix. And I'm sure there will be fixes as we progress here. I also considered moving the code here and using `go run` but I also use go 1.22 to build so that might be an issue for some folk (not sure?). I could expand this to prompt if the user wants the tool to be downloaded/installed and include a digest or similar.
This will be run only locally, the CI has the latest version included in the appstudio-util image...

Reply:
One small affordance I did is to do a sha256sum over the downloaded file...